README.md

```diff
@@ -22,20 +22,56 @@ Then you can install `fpm`:
 gem install --no-ri --no-rdoc fpm
 ```
 
-### install yarn for build dashboard
+### run build tool script:
+The version of APISIX is hard-code in `run.sh` now, you can change it by yourself.
+
+```
+./run.sh
+```
+
+
+## build apisix-dashboard RPM for CentOS 7
+
+### install Yarn、nodejs
 ```
 curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | sudo tee /etc/yum.repos.d/yarn.repo
-curl -sL https://rpm.nodesource.com/setup_12.x | sudo bash -
-sudo yum clean all && sudo yum makecache fast
-sudo yum install -y nodejs
-sudo yum install yarn
+curl --silent --location https://rpm.nodesource.com/setup_10.x | sudo bash -
+sudo yum install –y nodejs yarn
 ```
 
-### run build tool script:
-The version of APISIX is hard-code in `run.sh` now, you can change it by yourself.
+### install golang
+```
+wget https://dl.google.com/go/go1.15.2.linux-amd64.tar.gz 
+tar -xzf go1.15.2.linux-amd64.tar.gz
+sudo mv go /usr/local
+```
 
+### adjust the Path Variable for golang
+appending the following line to the `/etc/profile` file
 ```
-./run.sh
+export GO111MODULE=on
+export GOROOT=/usr/local/go 
+export GOPATH=$HOME/gopath
+export PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+```
+
+### load the new PATH environment variable with the following command:
+```
+source /etc/profile
+```
+
+### create the workspace directory for golang
+```
+cd ~
+mkdir gopath
+```
+
+### run build tool script for apisix-dashboard (must use root privileges):
+```
+./dashboard-rpm.sh
 ```
 
-The RPM package will be in `/tmp/rpm`.
+### after you installed the rpm of apisix-dashboard, run apisix-dashboard like this:
+```
+sudo manager-api -p /usr/local/apisix/dashboard/
```

Comment:
@idbeta since this command only read files under `/usr/local/apisix/dashboard`. So do we need to use `sudo` here?

Reply:
yes, otherwise it will prompt error
```panic: couldn't open sink "/usr/local/apisix/dashboard/logs/error.log": open /usr/local/apisix/dashboard/logs/error.log: permission denied
```

Reply:
> yes, otherwise it will prompt error
> 
> ```
> panic: couldn't open sink "/usr/local/apisix/dashboard/logs/error.log": open /usr/local/apisix/dashboard/logs/error.log: permission denied
> ```

@idbeta ok, my fault. I forgot the command would generate log files under `/usr/local/apisix/dashboard/logs/` directory. Thanks for explanations.